fastlane/Fastfile

```diff
@@ -111,9 +111,15 @@ lane :carthage_bootstrap do
   sh("cd ..; echo 'BUILD_LIBRARY_FOR_DISTRIBUTION=YES'>/tmp/config.xcconfig; XCODE_XCCONFIG_FILE=/tmp/config.xcconfig carthage bootstrap --platform iOS --new-resolver --no-use-binaries --cache-builds; rm /tmp/config.xcconfig")
 end
 
+desc "Builds the project for testing"
+lane :build do
+  scan(project: "StreamChat.xcodeproj", scheme: "StreamChat", build_for_testing: true, skip_testing: true)
```

Comment:
You don't need `skip_testing` flag, it's:
> Array of strings matching Test Bundle/Test Suite/Test Cases to skip
`build_for_testing` makes sure no tests are run anyway

Reply:
That's why CI failed :D

Reply:
Yep, I was too hurry to open to PR before lunch #hungry